fms_fsdp/utils/dataloader_utils.py

```diff
@@ -20,8 +20,6 @@ class SteadyCounter(torch.utils.data.IterableDataset):
         # Spit out incremental counts of constant length l, modulo vocab size v
         def __init__(self, l, v):
             self.i = 0
-            self.rank = 0
-            self.worldsize = 1
```

Comment:
Noting that we will need `rank` and `worldsize` if we ever want to wrap `SteadyCounter` in any of the existing non-dummy dataset wrappers